ui/components/multichain/app-header/app-header.js

```diff
@@ -23,10 +23,8 @@ import {
   BLOCK_SIZES,
   DISPLAY,
```

Comment:
While we are here we could update these deprecated consts to enums

Reply:
Updated https://github.com/MetaMask/metamask-extension/pull/19514/commits/84530cab2ebdf1edebbfae3bc1719b32efc42e63